src/mrack/outputs/ansible_inventory.py

```diff
@@ -106,12 +109,12 @@ class AnsibleInventoryOutput:
     files and information in DB.
     """
 
-    def __init__(self, config, db, metadata, path=DEFAULT_INVENTORY_PATH):
+    def __init__(self, config, db, metadata, path):
```

Comment:
I *think* you need to pass `path=None`, so `DEFAULT_INVENTORY_PATH` can be used. (I didn't test it)